packages/ecs-agent/0001-bottlerocket-default-filesystem-locations.patch

```diff
@@ -1,17 +1,19 @@
-From c6cbf43d62a587b7a7672b726c4513005c9b77e8 Mon Sep 17 00:00:00 2001
+From 5c3c33e52192080eeeb5527a9ef4b4afec01d4d1 Mon Sep 17 00:00:00 2001
 From: Samuel Karp <skarp@amazon.com>
 Date: Wed, 8 Jul 2020 11:18:35 -0700
-Subject: [PATCH 1/5] bottlerocket: default filesystem locations
+Subject: [PATCH] bottlerocket: default filesystem locations
```

Comment:
In general, I prefer to regenerate the full patch series rather than just one patch.